pkg/cmd/pulumi/new.go

```diff
@@ -201,7 +204,7 @@ func runNew(ctx context.Context, args newArgs) error {
 	} else if len(templates) == 1 {
 		template = templates[0]
 	} else {
-		if template, err = chooseTemplate(templates, opts); err != nil {
+		if template, err = args.chooseTemplate(templates, opts); err != nil {
```

Comment:
This is a slightly terrifying change in terms of a potential nil reference panic. I _think_ I checked and it should be fine - but may I ask the reviewer to double-check that we'll never pass `nil` here from the CLI binary (not tests)?

Reply:
Yeah, I can't find any instances other than `newNewCmd`.